File: cs/proxy/hall/HallGeneralizeNotify.cs
/// <summary>
/// 
/// </summary>
namespace org.zgl.service.client.hall{
	[Proxy(gameId = -10)]
	public interface HallGeneralizeNotify : INotify { 
		/// <summary>
		/// 如果玩家在线 通知推广人有玩家填了你的邀请码
		/// </summary>
		void notifyInvite(GeneralizeAwardDto dto);
	}
}

File: componet/src/main/java/org/zgl/service/client/hall/HallGeneralizeNotify.java
package org.zgl.service.client.hall;

import org.zgl.dto.clinet.hall.GeneralizeAwardDto;
import org.zgl.proxy.MethodDesc;
import org.zgl.proxy.rule.INotify;

/**
 * @author： 猪哥亮
 * @创建时间： 2018/8/22
 * @文件描述：
 */
public interface HallGeneralizeNotify extends INotify {
    @MethodDesc("如果玩家在线 通知推广人有玩家填了你的邀请码")
    void notifyInvite(GeneralizeAwardDto dto);
}
